ci-operator/step-registry/ipi/conf/aws/route53/ipi-conf-aws-route53-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+
+# subnet, AZs and hosted zone
+allsubnetids="${SHARED_DIR}/allsubnetids"
+hostedzoneid="${SHARED_DIR}/hostedzoneid"
+availabilityzones="${SHARED_DIR}/availabilityzones"
+if [ ! -f "${allsubnetids}" ] || [ ! -f "${availabilityzones}" ] || [ ! -f "${hostedzoneid}" ]; then
+    echo "File ${allsubnetids} or ${availabilityzones} or ${hostedzoneid} does not exist."
+    exit 1
+fi
+
+echo -e "subnets: $(cat ${allsubnetids})"
+echo -e "AZs: $(cat ${availabilityzones})"
+echo -e "hosted zone: $(cat ${hostedzoneid})"
+
+CONFIG_ROUTE53_HOSTEDZONE="${SHARED_DIR}/install-config-route53.yaml.patch"
+cat > "${CONFIG_ROUTE53_HOSTEDZONE}" << EOF
+publish: External
```

Comment:
suggest to remove this irrelevant setting

Reply:
Done